doc/changes/latest.inc

```diff
@@ -113,6 +113,8 @@ Enhancements
 
 - `mne.viz.plot_drop_log` and :meth:`mne.Epochs.plot_drop_log` now omit displaying the subject name in the title if ``subject=None`` is passed (:gh:`9015` by `Richard Höchenberger`_)
 
+- `mne.io.meas_info.anonymize_info` now anonymize also sex and hand fields when keep_his is False (:gh:`8130` by `Rotem Falach`_)`
```

Comment:
once you implemented my requests from https://github.com/mne-tools/mne-python/pull/9103#issuecomment-801133563 you will need to adjust this line:

```suggestion
- `mne.io.meas_info.anonymize_info` now anonymize also sex and hand fields when keep_his is False (:gh:`8130` by |Rotem Falach|_)`
```